tendrl/node_agent/flows/generate_journal_mapping/__init__.py

```diff
@@ -9,7 +9,8 @@ class GenerateJournalMapping(flows.BaseFlow):
     def run(self):
         # Generate the journal mapping for the nodes
         mapping = utils.generate_journal_mapping(
-            self.parameters['Cluster.node_configuration']
+            self.parameters['Cluster.node_configuration'],
+            integration_id=self.parameters.get("TendrlContext.integration_id")
```

Comment:
Its an optional parameter. It should be passed if cluster is already existing and for additional disks we want a journal mapping. If not passed normal mapping would be done as usual. In case of existing cluster, while mapping it would try to check if SSDs allow more journals to be created on them.